lib/documents/schemas/marine_equipment_approved_recommendations.json

```diff
@@ -0,0 +1,102 @@
+{
+  "pre_production": true,
+  "content_id": "49f47764-2b1b-4b0d-9164-4aa6b42a8b63",
+  "base_path": "/marine-equipment-approved-recommendations",
+  "format_name": "UK Approved Recommendations – Marine Equipment",
+  "name": "UK Approved Recommendations – Marine Equipment",
+  "description": "Find recommendations on how to interpret and apply the Merchant Shipping (Marine Equipment) Regulations 2016 as amended.",
+  "filter": {
+    "format": "marine_equipment_approved_recommendation"
+  },
+  "editing_organisations": [
+    "23a24aa8-1711-42b6-bf6b-47af0f230295"
+  ],
+  "organisations": [
+    "23a24aa8-1711-42b6-bf6b-47af0f230295"
+  ],
+  "related": [
+    "e234eed3-8ab9-4c45-af63-1fca0029e165",
+    "03f48130-94ba-47e5-8112-e9af5e018e2d"
+  ],
+  "signup_content_id": "512a14c1-260a-4c4e-8de8-643eadf7905c",
+  "signup_copy": "You’ll get an email each time a new UK Approved Recommendation is published.",
+  "subscription_list_title_prefix": "UK Approved Recommendations for marine equipment",
+  "summary": "<p>Find recommendations on how to interpret and apply the Merchant Shipping (Marine Equipment) Regulations 2016 as amended.</p>",
+  "document_noun": "recommendation",
+  "facets": [
+    {
+      "key": "category",
+      "name": "Category",
+      "short_name": "Recommendation type",
+      "type": "text",
+      "preposition": "With recommendation type",
+      "display_as_result_metadata": true,
+      "filterable": true,
+      "allowed_values": [
+        {
+          "label": "Life-saving appliances",
```

Comment:
Should we have a dash in `Life-saving`?

Reply:
This came from their requirements documentation - we can double check with MCA when we run a test in integration?